modules/persistent-storage-csi-group-snapshots-create.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-create_{context}"]
+= Creating a volume group snapshot
+
+When you create a `VolumeGroupSnapshot` object, {product-title} creates a volume group snapshot.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* A persistent volume claim (PVC) created using a CSI driver that supports `VolumeGroupSnapshot` objects.
+* A storage class to provision the storage back end.
+* Administrator has created the `VolumeGroupSnapshotClass` object.
+* No pods are using the PVC that you want to take a snapshot of.
++
+[WARNING]
+====
+Creating a volume snapshot of a PVC that is in use by a pod can cause unwritten data and cached data to be excluded from the snapshot. To ensure that all data is written to the disk, delete the pod that is using the PVC before creating the snapshot.
+====
+
+.Procedure
+
+To create a volume group snapshot:
+
+. Locate (or create) the PVCs that you want to include in the volume group snapshot:
++
+[source, terminal]
+----
+$ oc get pvc
+
+NAME        STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+pvc-0       Bound     pvc-a42d7ea2-e3df-11ed-b5ea-0242ac120002   1Gi        RWO           48s
+pvc-1       Bound     pvc-a42d81b8-e3df-11ed-b5ea-0242ac120002   1Gi
+----
++
+This example uses two PVCs
+
+. Label the PVCs to belong to a snapshot group by running the following commands:
++
+[source, terminal]
+----
+$ oc label pvc pvc-0 group=myGroup
+persistentvolumeclaim/pvc-0 labeled
+
+$ oc label pvc pvc-1 group=myGroup
+persistentvolumeclaim/pvc-1 labeled
```

Comment:
Two commands need to be displayed separately. In this case, I guess we can use a placeholder with the command syntax.